__tests__/api.tests/general.test.js

```diff
@@ -18,4 +18,112 @@ describe('GET topicccccs', () => {
             expect(body.msg).toBe('Bad api endpoint');
         });
     })
+})
+
+describe('topics', () => {
+    test('GET /api/topics returns with status code 200 and all the topics', () => {
+        return request(app).get('/api/topics').expect(200).then(({body: topics}) => {
+            topics.forEach(topic => {
+                expect(topic).toEqual(expect.objectContaining({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                }))
+            })
+        });
+    })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('200: GET /api/articles/:article_id/comments responds status 200 with an array of comment objects for a specific article, ordered last comment first', () => {
+        return request(app).get('/api/articles/3/comments').expect(200).then(({body: comments}) => {
+            expect(comments.comments).toHaveLength(2);
+            expect(comments.comments).toBeSortedBy('created_at', {descending: true});
+            comments.comments.forEach(comment => {
+                expect(comment).toEqual(expect.objectContaining({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
Would you expect any number for the article id in this test? As you are only expecting comments for the article with id of 3, I think this is a good place to assert this and ensure they are the only comments present